### ISSUE TITLE
Error handling

### ISSUE THREAD

**Issue by desainishit** [aerkalov/ebooklib#161]
I am trying to read an ebook which has `<dc:title>` tag missing in content.opf file and as a result, `read_epub()` function fails. Is there a way to make it more fault tolerant for this?

**Comment by aerkalov** [comment]
For sure. Could you send me your EPUB if it is not a problem to test it? I assume it is this piece of code but in theory it should not break.

` 
titles = self.book.get_metadata('DC', 'title')
if len(titles) > 0:
    self.book.title = titles[0][0]
`

**Comment by desainishit** [comment]
Exactly! That's the one that's breaking. Here's the file that's causing it. https://files.fm/u/g2jk3h46

**Comment by aerkalov** [comment]
There is default value for metadata now (empty list). There should be also some kind of error when user is trying to use non existing namespace, will check that later.

**Comment by desainishit** [comment]
That was quick! Thank you.

**Comment by dwprice56** [comment]
This fixed the problem for me (file epub.py,  line 1406):

        try:
            titles = self.book.get_metadata('DC', 'title')
            if len(titles) > 0:
                self.book.title = titles[0][0]
        except KeyError:
            pass
